docs/nodes/full-node/update-fullnode-with-new-releases.md

```diff
@@ -1,47 +1,72 @@
 ---
-title: "Update Fullnode With New Devnet Releases"
+title: "Update Fullnode With New Releases"
 slug: "update-fullnode-with-new-devnet-releases"
 sidebar_position: 11
 ---
 
 # Update Fullnode With New Releases
 
-When `devnet` is wiped and updated with newer versions, you will need to update your fullnode as well. If you do not, your fullnode will not continue to synchronize with the network. To update your fullnode, follow these steps:
+This document outlines the process for updating your fullnode. For fullnodes running in `devnet`, an additional data wipe step is required as `devnet` is wiped on every release.
 
 ## If you built the fullnode from aptos-core source code
 
-1. Shutdown your fullnode.
+1. Stop your fullnode by running the below command:
 
-2. Delete the data folder (the directory path is what you specified in the configuration file, e.g., `fullnode.yaml`).
+```bash
+cargo stop aptos-node
+```
 
-   - The default data folder is `/opt/aptos/data`.
+2. For users of the Rust binary, pull the latest release appropriate for your network (`devnet`, `testnet`, or `mainnet`):
+
+```bash
+git checkout [network_branch] && git pull
+```
+
+Replace `[network_branch]` with `devnet`, `testnet`, or `mainnet` as applicable, and rebuild the binary.
+
+3. If your fullnode is running in `devnet`, follow the additional steps in the [Additional data wipe steps for `devnet`](#additional-data-wipe-steps-for-devnet) section below.
```

Comment:
`mainnet` needs to upgrade to release branches rather than `mainnet`, since `mainnet` usually lags behind for framework changes right?

Reply:
I think we can probably keep it simple for now and point to the network branches (i.e., leave it as it is). Looking at the various tests (e.g., the fullnode sync jobs and the commit checker), the network branches are good enough 😄